### PR TITLE
App/request referral

### PR DESCRIPTION
**Pull request by FrozenKiwi**
Embedded prefinery form for registering sign-up links.

Not sure if we'll actually push this, but we need something

image

### CODE REVIEW

File: apps/site-landing/src/containers/ApplyBeta/index.tsx [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+          <FormattedMessage {...translations.whatToExpectContent} />
+        </Message.Content>
+      </Message>
+      <div className='prefinery-form-embed'>
+      </div>
+      <Helmet>
+        <script src="https://widget.prefinery.com/widget/v2/5jn8p69h.js" defer></script>
+      </Helmet>
+    </div>
+  );
```
Comment (FrozenKiwi) [inline apps/site-landing/src/containers/ApplyBeta/index.tsx:68]: This could clearly be a lot better

Comment (FrozenKiwi) [reply apps/site-landing/src/containers/ApplyBeta/index.tsx:68]: The text really needs to be prettied up, but I just couldn't put my head in writer mode when I wrote it.

File: libs/site-base/lang/en.json [hunk 2]
```diff
@@ -235,6 +235,10 @@
     "defaultMessage": "Create Account",
     "description": "app.account.create.form.buttonCreate: Button for the create account form page"
   },
+  "bIBRtw": {
+    "defaultMessage": "Join our Beta",
+    "description": "Super-title on request access page"
```
Comment (MarieTheCoin) [inline libs/site-base/lang/en.json:240]: Is this a new type of title?

**Review by MarieTheCoin — APPROVED** [review]

File: libs/site-base/lang/en.json (hunk 2 above)
Comment (FrozenKiwi) [reply libs/site-base/lang/en.json:240]: Err... no - just short-hand for "Above the Title".  Looks a little silly now I read it

**Pull request merged**